commands/cli.go

```diff
@@ -19,17 +22,32 @@ func CreateCli(version string) *cli.App {
 	app.HelpName = app.Name
 	app.Author = "Gruntwork <www.gruntwork.io>"
 	app.Version = version
-	app.Usage = "A CLI tool to cleanup AWS resources (EC2). THIS TOOL WILL COMPLETELY REMOVE ALL RESOURCES AND ITS EFFECTS ARE IRREVERSIBLE!!!"
+	app.Usage = "A CLI tool to cleanup AWS resources (ASG, ELB, ELBv2, EBS, EC2). THIS TOOL WILL COMPLETELY REMOVE ALL RESOURCES AND ITS EFFECTS ARE IRREVERSIBLE!!!"
+	app.Flags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "exclude-region",
+			Usage: "regions to exclude",
+		},
+	}
 	app.Action = errors.WithPanicHandling(awsNuke)
 
 	return app
 }
 
 // Nuke it all!!!
 func awsNuke(c *cli.Context) error {
-	logging.Logger.Infoln("Retrieving all active AWS resources")
+	regions := aws.GetAllRegions()
+	excludedRegions := c.StringSlice("exclude-region")
+
+	for _, excludedRegion := range excludedRegions {
+		if !collections.ListContainsElement(regions, excludedRegion) {
+			fmt.Println(excludedRegion + "is not a valid AWS Region")
+			return InvalidFlagError{}
```

Comment:
Instead of the error message in a println and a separate error with no info, update `InvalidFlagError` to have the flag name and value as fields and to have an `Error()` method that clearly shows them to the user.